<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="SubjectsTab" name="subjectsTab">
       <attribute name="title">
        <string>Mati√®res</string>
       </attribute>
      </widget>
      <widget class="TeachersTab" name="teachersTab">
       <attribute name="title">
        <string>Enseignants</string>
       </attribute>
      </widget>
      <widget class="GroupsTab" name="groupsTab">
       <attribute name="title">
        <string>Groupes</string>
       </attribute>
      </widget>
      <widget class="OptionsTab" name="optionsTab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
      </widget>
      <widget class="ComputationTab" name="computationTab">
       <attribute name="title">
        <string>Calcul</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubjectsTab</class>
   <extends>QWidget</extends>
   <header>SubjectsTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TeachersTab</class>
   <extends>QWidget</extends>
   <header>TeachersTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionsTab</class>
   <extends>QWidget</extends>
   <header>OptionsTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComputationTab</class>
   <extends>QWidget</extends>
   <header>ComputationTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupsTab</class>
   <extends>QWidget</extends>
   <header>GroupsTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
